db/migrate/20240814051005_drop_users.rb

```diff
@@ -1,5 +1,5 @@
 class DropUsers < ActiveRecord::Migration[7.1]
   def change
-    drop_table :users
+    drop_table :users, if_exists: true
```

Comment:
This is not required, there should be an  existing migration which creates users table, else this migration is not required.

Reply:
drop_user migration file is changed to its original.